forge/test/mlir/llama/test_llama_inference.py

```diff
@@ -126,3 +127,141 @@ def test_llama_inference_cache_cpu():
     # Generated text
     generated_text = tokenizer.decode(generated_tokens[0], skip_special_tokens=True)
     print(generated_text)
+
+
+@pytest.mark.xfail()
+def test_llama_inference_no_cache_tt():
```

Comment:
Let's outline in test name that it's decode on tt, prefill is on CPU

Reply:
Sure

Reply:
Renamed 
`test_llama_inference_no_cache_tt` -> `test_llama_prefill_on_cpu_decode_on_tt_no_cache`
`test_llama_inference_cache_tt` -> `test_llama_prefill_on_cpu_decode_on_tt_cache`